tests/fsharp/SDKTests/tests/DefaultFSharpCore - IsPreviewSdkEnablePreviewCheck.proj

```diff
@@ -0,0 +1,21 @@
+<Project ToolsVersion="4.0" DefaultTargets="Test" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+  <!-- Verifies that in a preview sdk with preview check enabled it selects the shipped fsharp.core package -->
```

Comment:
`shipped fsharp.core package` => `preview fsharp.core package`

Test is good, comment is incorrect.